usecase/transport/test/test_usecase.py

```diff
@@ -0,0 +1,72 @@
+
+import unittest
+from datetime import datetime, timedelta
+from freezegun import freeze_time
+import os
+import json
+
+from usecase.usecase import Reply
+from usecase.transport import Transport
+from services.weatherAPI import WeatherAdapter
+from services.preferences import PrefService, PrefJSONRemote
+from services.maps import GeocodingService, MapService
+from services.vvs import VVSService, VVSEfaJSONRemote
+from services.weatherAPI.test import WeatherMock
+#from services.maps.test import MapMockRemote, GeocodingMockRemote
+#from services.vss.test import VVSMockRemote
+
+class TestTransport(unittest.TestCase):
+    dhbw = [48.773563, 9.170963]
+    mensa = [48.780834, 9.169989]
+            
+    @classmethod
+    def setUpClass(self):
+        self.pref_service=PrefService(PrefJSONRemote()),
+        self.map_service=MapService.instance(),
+        self.vvs_service=VVSService.instance(VVSEfaJSONRemote.instance()),
+        self.geo_service=GeocodingService.instance(),
+        self.wea_service=WeatherAdapter.instance()
+        '''    
+        if 'DONOTMOCK' in os.environ:
+            self.pref_service=PrefService(PrefJSONRemote()),
+            self.map_service=MapService.instance(),
+            self.vvs_service=VVSService.instance(VVSEfaJSONRemote.instance()),
+            self.geo_service=GeocodingService.instance(),
+            self.wea_service=WeatherAdapter.instance()
+        else:
+            print("Mocking remotes...")
+            self.pref_service=PrefService(PrefJSONRemote()),
+            self.map_service=MapService.instance(),
+            self.vvs_service=VVSService.instance(VVSEfaJSONRemote.instance()),
+            self.geo_service=GeocodingService.instance(),
+            self.wea_service=WeatherAdapter.instance()
+        '''
+        self.use_case = Transport()
+        self.use_case.set_services()
+
+    # This test only works correct when the weather is good
+    @freeze_time("2020-03-26 21:30:00")

```

Comment:
I am freezing the time so that the routes are calculated correctly, since some routes are not available if there is not enough time left. However, this is still useful for the weather. Thanks

Reply:
Ah I see!